sql/nysdec_solidwaste.sql

```diff
@@ -30,7 +30,6 @@ SET hash =  md5(CAST((t.*)AS text)),
                     WHEN activity_desc LIKE '%RHRF%' THEN 'Recyclables Handling and Recovery'
                     WHEN activity_desc LIKE '%medical%' THEN 'Regulated Medical Waste'
                     WHEN activity_desc LIKE '%Transfer%' THEN 'Transfer Station'
-                    WHEN activity_desc LIKE '%Composting%' THEN 'Composting'
```

Comment:
Why is this row deleted? I checked through the dataset and there are 10 records for compose
@baolingz

Reply:
because there is a identical row of codes in line 28.

Reply:
oh ok